.ci/util.sh

```diff
@@ -25,6 +25,15 @@ function getCheckstylePomVersion {
   getMavenProperty project.version
 }
 
+function getCheckstylePomVersionWithoutSnapshot {
+  getCheckstylePomVersion | sed "s/-SNAPSHOT//"
+}
+
+function getCheckstylePomVersionWithoutSnapshotWithXmlstarlet {
+  xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
+                             -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//"
```

Comment:
Please make indentation only +2 from initial command, like:
```bash
  xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
    -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//"

````

Reply:
Done. I will keep that in mind for the future.